Framework/Algorithms/src/SolidAngle.cpp

```diff
@@ -55,13 +124,25 @@ void SolidAngle::init() {
                   "The index of the last spectrum whose solid angle is to be "
                   "found (default: the\n"
                   "last spectrum in the workspace)");
+
+  const std::vector<std::string> methods{"GenericShape", "Rectangular",
+                                         "VerticalTube", "HorizontalTube",
+                                         "VerticalWing", "HorizontalWing"};
+  declareProperty(
```

Comment:
The details of what everything does will be easier to read in the `.rst` file. Here they get stuffed in a tiny box in a table.